### DISCUSSION TITLE
posts/pterodactyl-game-server/

### DISCUSSION THREAD

**Discussion by giscus** [techno-tim/techno-tim.github.io#204] [Comments]
# posts/pterodactyl-game-server/

Documentation for Techno Tim YouTube videos for all your copy pasta needs!

https://techno-tim.github.io/posts/pterodactyl-game-server/

**Comment by FDEthan** [comment]
Hi Tim

Would you be able to share your reverse proxy setting for this.
I have tried all sorts of things to get this working.

Thank you

**Reply by timothystewart6** [reply]
Hey!  I don't have one specific to pterodactyl, but here is my traefik config for most services https://github.com/techno-tim/launchpad/tree/master/docker/traefik

**Comment by FDEthan** [comment]
Thank you!

**Reply by Krieger-Papagai** [reply]
Did you manage to get this working? I have been struggling with this as well.

**Comment by Choque** [comment]
I'm currently trying to get both the panel and the wings running on the same server (because...why would I need two servers for that :sweat_smile:) by combining the two docker-compose files but I somehow can't get it to work (also behind traefik that is).
I'm also not really sure which if any ports I need to expose. I usually just add the one port that is required for most services via the traefik.loadbalancer label so for, for example Heimdall, my docker-compose.yml has the following label
```yaml
- traefik.http.services.heimdall.loadbalancer.server.port=80
```

Since the video is a simplified version (not using traefik), are you exposing any ports in your "production" deployment of pterodactyl with traefik?

**Comment by CKMartens** [comment]
Hello Tim,

a small error has crept into your documentation. Under 'Create a User' you forgot the last letter in make It should be docker-compose run --rm panel php artisan p:user:make

**Reply by timothystewart6** [reply]
Thank you!  Fixed!

**Comment by richardvrusso** [comment]
Hi everyone,
I'm following along with two ubuntu server VMs in proxmox. So everything is fresh installs. I can't access the Panel over https but I can get it on http. However after I login it keeps bouncing over to https and then goes 404. If I remove the S then the page loads. I started going down the google search rabbithole but nothing concrete. Does anyone have a good URL they can link me to?

Regards, Rick

**Comment by skylanoris** [comment]
Hello, 
i have one problem i cant create node 
the error:  The fqdn could not be resolved to a valid IP address.

**Reply by poisonWinslow** [reply]
I was having this issue. If you are using pihole & traefik to have wildcard SSL certs within your local network like I was, you'll need to change the DNS server of the server running the pterodactyl panel to point at pihole. You'll need to look up how to change your DNS depending on which distro your using

**Reply by poisonWinslow** [reply]
I should add, you'll also need to include your node in your traefik config.yml & will need a local DNS entry within pihole similar to your other local dns setups

**Comment by njalooo** [comment]
Hey, the docker image for pterodactyl seems to be broken, inside the container netstat showed nothing listening on port 443.
I would love to see an update to this guide in the future, especially with it beeing set up behind traefik on one machine.

**Reply by njalooo** [reply]
Aah nevermind, i figured out that you need to send traffic to Port 80, not 443 when using Traefik.

**Reply by njalooo** [reply]
Here are te labels to configure traefik, also don't expose any ports.

labels:
      - "traefik.enable=true"
      - "traefik.http.routers.pterodactyl.entrypoints=https"
      - "traefik.http.routers.pterodactyl.rule=Host(`pterodactyl.njal.ch`)"
      - "traefik.http.routers.pterodactyl.tls=true"
      - "traefik.http.routers.pterodactyl.service=pterodactyl"
      - "traefik.http.services.pterodactyl.loadbalancer.server.port=80"
      - "traefik.docker.network=traefik"

**Comment by effelle** [comment]
Hi Tim!
Thanks for this guide. 
I was used to Nginx but lately I'm only using a Cloudflare tunnel due to my ISP blocking ports 80 and 443. What should I change to have Pterodactyl just local? 

Thanks in advance for your reply.

**Comment by pixelwave** [comment]
I am trying to merge the knowledge from both your videos together (about how to install traefik with wildcard certificates and this one), with some other sources sprinkled in. But I am having huge issues though. Would be awesome to have one video setting up a gameserver from start to finish with all necessary VMs, docker, traefik, portainer.

Right now I am stuck since the panel uses the same 443 port as the traefik container ("Bind for 0.0.0.0:443 failed: port is already allocated"). -.-

**Comment by dbeckler** [comment]
I am having the following issue with a new install and using the above docker-compose.
SQLSTATE[HY000] [1045] Access denied for user 'pterodactyl'@'172.20.0.5' (using password: YES

Has anyone seen this issue?  I have not changed anything other than the password.  I have tried this on a Fedora server and Ubuntu server and see the same issue

**Reply by the0ckid82** [reply]
having the same issue as well, have you figured out a solution ?

**Reply by Berleos** [reply]
The one deal that worked for me was to run the docker-compose file and when everything was running, restart MariaDB only and that should start normally afterwards.

**Comment by allen-mckenzie** [comment]
Running into an issue with this binding the volumes directory. I've tried changing the UID and GUID used in the compose file for the wings but when you install a server it pulls the image and then says that the volume can't be found and throws an error about the bind

container@pterodactyl~ Error Event [290f1afb-ec64-48c8-a73b-6485f54ae532]: environment/docker: failed to run pre-boot process: environment/docker: failed to create container: Error response from daemon: invalid mount config for type "bind": bind source path does not exist: /var/lib/pterodactyl/volumes/c2892941-51b9-4910-929b-6bda356ca375

The crazy part is that everything else is working fine and the volume directory is present and the permissions seem to be correct as well but I'm lost as to why this is failing. I've uninstalled and reinstalled docker 4 times now, purged the images and compose scripts and started this process fresh more times than I can count thinking that maybe I missed something along the way but still get the same result here. 

I initially ran into issues with the default ubuntu firewall and had to remove all of that because the ports wouldn't respond at all. I'll lock that down again once I get things working but otherwise I'm just stuck here at the moment.

**Reply by allen-mckenzie** [reply]
So, the rabbit hole gets deeper. I have even tried running this using the traefik configuration and with that I end up with a CORS issue that I can't resolve at the moment either. Effectively, I'm finding this whole tutorial to not work and not sure where things are going amiss here. 

I setup the panel to run on a raspberry pi I have lying around and the panel works fine but when the wings agent is connected and showing green in the panel the wings agent is unable to install a server.

So then if I try removing the pi out of the equation and start with a brand new install of ubuntu on an old pc and try running this I can get the panel running but then it can't connect to the wings agent due to a CORS issue.

I even tried following another guide to try and work around this issue entirely from https://github.com/EdyTheCow/docker-pterodactyl but this one has a whole other pile of problems and when I correct my way through those and get traefik to see the panel it ends up having the same CORS issue even though I'm using 2 separate domains.

Honestly, I've had better luck just standing up separate docker containers for game servers instead of trying to get a management panel in place like this within docker and I just don't know where to turn at this point.

Do you need 2 separate servers to run this thing? If so, that would be great to know.
If you use traefik so that you can have SSL certs applied to everything then why can't you use a subdomain for a wing?

Example:

domain.com running traefik with a wildcard subdomain *.domain.com to issue SSL certs to
panel.domain.com running the pterodactyl panel service
wing0.domain.com running the wings agent

wing0.domain.com results in CORS error ( Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://wing0.domain.com/api/system. (Reason: CORS request did not succeed). Status code: (null). )

Other issue arise with this as well where panel.domain.com throws error in the console for 500 errors and if you try to edit previous node for anything you get a 500 error as well.

It has been one hell of a struggle so far I tell ya and I have made 0 progress with this and honestly ready to call it quits. Not even sure if anyone can see my questions here or if they have run into similar issues with this.  Any insight or possible solutions I could try are more than welcome here as well.

**Reply by despwndence** [reply]
so I figured it out you need to use the cors and security middleware but, you need to point the service to http://yourip:yourport emphasis HTTP. The panel node should be setup like described with 443 as the daemon port behind  a proxy. also, i would  make sure you specify both your trusted proxy and allowed orgins in the wing config.yml.

**Reply by Swedorican-Design** [reply]
having the exact same issue with CORS and it's driving me nuts.  Could you give a better explanation or an example of the lines you added and where?

**Reply by OneBiteAidan** [reply]
@Swedorican-Design Despwndence is saying that in your Traefik config.yml you have to set your wings service server endpoint as http://your_wings_ip:443
Here's a snippet of my Traefik's [config.yml](https://pastebin.com/THB9u3z8)

**Comment by marcodantonio** [comment]
In my case panel and Wings are on the same machine, in two different containers. The way docker-compose is set up both use port 443 and of course the containers don't start because port 443 can't be associated with two containers at the same time. how can I solve this?

**Reply by dbeckler** [reply]
@marcodantonio - Change your port declaration to something like the below. 

The first number (before the colon "11443:) is the port you are using on the machine and it has to be unique.  The number after the colon (443) is the port the program is using inside a container.  As you run more containers on the same machine you will have to find free port numbers to use so they don't overlap.

ports:
      - "11080:80"
      - "11443:443"

**Reply by marcodantonio** [reply]
I know this. I don't know if the problem is for Wings. Let me explain: in the config.yaml file as per the video I set it to use port 443. But in theory, there is no need to open the port since the two containers are on the same network, right? Then how come I keep getting the red heart?

**Reply by gadget78** [reply]
The reason you get red heart. As it needs to still talk to the wings/deamon manager on port 443 too, which clash.. and as 443 is default port for https , you will have issues, unless you do some port forwarding stuff... 

the best solution for this to be on the same "machine" ,
is to use proxmoxx (or other such type like VM ware).

you can make 3 "containers"
1 for the panel,
1 for node/wings/deamon, (whatever you wanna call it) so they can have distict IP/Ports
then 1 for ngnix, i prefer ngnix proxy manager ..
then use ngnix as you main 80/443 landing place (NPM) so you can use it for lets encrypt, distribute web calls...

good place for preconfigured containers is https://tteck.github.io/Proxmox/ (for Debian and NPM)

**Comment by jlengelbrecht** [comment]
I am having an issue with the deployment of the panel. I added the server to my traefik config in order to connect via a custom domain. When using the domain I designated I get a bad gate way issue. when connecting to to the panel via the IP it will connect and i can sign in. However when I navigate to the setting tab I get a connection refused error. Not really sure what I am doing wrong.

**Comment by ncastaldi** [comment]
Hoping I could get some guidance on setup. I'm struggling to get the panel stood up to start and am getting the following in the panel logs.

Thanks in advance!

-------------------LOG OUTPUT BELOW-------------------

ln: /app/storage/logs/logs: File exists
external vars exist.
Checking if https is required.
Using nginx config already in place.
No letsencrypt email is set
Checking database status.
Waiting for database connection...
database (172.50.0.4:3306) open
Migrating and Seeding D.B
In Connection.php line 760:
                                                                               
  SQLSTATE[HY000] [1045] Access denied for user 'pterodactyl'@'172.50.0.5' (u  
  sing password: YES) (SQL: select * from information_schema.tables where tab  
  le_schema = panel and table_name = migrations and table_type = 'BASE TABLE'  
  )                                                                            
                                                                               
In Connector.php line 70:
                                                                               
  SQLSTATE[HY000] [1045] Access denied for user 'pterodactyl'@'172.50.0.5' (u  
  sing password: YES)                                                          
                                                                               
Starting cron jobs.
Starting supervisord.
2023-05-12 15:25:31,082 CRIT Server 'unix_http_server' running without any HTTP authentication checking

**Reply by ncastaldi** [reply]
I fixed my issue. Not entire sure how these two pieces are connected yet, but I completed the "mail" section at the top of the docker-compose, redeployed, and can now hit the sign in page in my browser. On to the next step!!!

**Comment by KittyFloof** [comment]
It doesn't work no matter what I do. I've followed so many different paths just to get absolutely nothing at all. its useless. Tim skips steps and brushes off important things and expects you to figure it out. I've been at this for 3 whole days with nothing to show for it. I've tried everything I could think of and nothing is working at all.

**Comment by theRealBassist** [comment]
I really wish I could get this working. I can't get Let's Encrypt to actually validate my domain, even though it works fine for everywhere else, I think that the Let's Encrypt issue is likely causing my connections issues because I can't even load the panel.

From what I've seen, the installation of this tool is godawful. I've spent over a day going between different install guides (even using several provided on their website) and searching around their Discord. From what I've seen the devs just refuse to help with any kind of installation problems and immediately close issues without offering any help or advice. When they do deign to reply, it's normally to insult users and tell them to stop trying to install their software.

I'm sure this is a tool that works fine, when you can get it working, but nothing surrounding this entire experience, including your advice here, has been helpful or positive.

**Reply by njalooo** [reply]
I got it working by giving up on https validation, instead i changed cloudflare to be my DNS server, and used their API keys in traefik with dns validation to get it working. The devs of let's encrypt beeing annoyed is understandable, since your issue likely lies with traefik or it's misconfiguration, not their service. Plus is, you can easily configure a server based dynamic DNS, by using another docker container for it.

**Reply by gadget78** [reply]
the best solution for this to be on the same "machine" ,
is to use proxmoxx (or other such type like VM ware).

you can make 3 "containers"
1 for the panel,
1 for node/wings/deamon, (whatever you wanna call it) so they can have distict IP/Ports
then 1 for ngnix, i prefer ngnix proxy manager ..
then use ngnix as you main 80/443 landing place (NPM) so you can use it for lets encrypt, distribute web calls...

good place for preconfigured containers is https://tteck.github.io/Proxmox/ (for Debian and NPM)

**Comment by gadget78** [comment]
i came here to ask/point out, why in your wings compose file do you have the image on a static version of 1.6.1 ? 
and not latest ?

so currently its on,
```
  wings:
    image: ghcr.io/pterodactyl/wings:v1.6.1
```
why not ,
```
  wings:
    image: ghcr.io/pterodactyl/wings:latest
```

is there a reason for this ?
as at some point in future this will break things,
Wasnt to breaking for me, but the reason i realised was because of login issues with Admin accounts, then realised i didn't have the latest software .. 
which was odd, as i had just installed it all .. 
thought i would just try flag this up for you here, not sure you or anyone reads these now ??

**Reply by jlengelbrecht** [reply]
Have you found a way to spin this up with the new panel and wings? I was able to get this to work a while back but it has since become unstable, and won't let you access the sever management area without crashing. The version Tim used also unfortunately can not be updated. Would love to reinstall the new version of this. Perhaps you have a updated docker-compose I could get from you?

**Reply by gadget78** [reply]
Yes, I used his scripts, and it worked OK.. 

Just Recently changed image used for wings to latest, and then rebuilt the container, within portainer, and all seems to be working still 100%
It's all on the same machine, running proxmox,
Utilising 3 containers;
1 for panel, 
1 for node/wings, and 
1 for ngnix proxy manager ...

**Reply by ncastaldi** [reply]
care to share your docker-compose(s)?

**Reply by gadget78** [reply]
i used the ones above, just with my domain names etc ... 
just followed his YouTube, and used NGBIX proxy manager to sort out all the incoming connections to 80 / 443 
to keep the SSL connectons :)

**Comment by junny2233** [comment]
Hello, 


I was searching around the community but i didn't find an answer so im going to ask here to see if someone can explain me what im doing wrong

im getting this error after following the tutorial https://technotim.live/posts/pterodactyl-game-server/ and https://www.youtube.com/watch?v=_ypAmCcIlBE
 have anyone be able to fix this issue? the panel container works perfectly fine but the wing is not working.

https://cdn.junny2233.xyz/mafE3/riZuteLU44.png/raw

`This software is made available under the terms of the MIT license.
The above copyright notice and this permission notice shall be included
in all copies or substantial portions of the Software.

 INFO: [Jul  2 22:42:31.868] loading configuration from file config_file=/etc/pterodactyl/config.yml
 INFO: [Jul  2 22:42:31.868] configured wings with system timezone timezone=UTC
 INFO: [Jul  2 22:42:31.868] configured system user successfully gid=redacted uid=redacted username=redacted
 INFO: [Jul  2 22:42:31.869] fetching list of servers from API
 INFO: [Jul  2 22:42:31.999] processing servers returned by the API total_configs=0
 INFO: [Jul  2 22:42:31.999] finished processing server configurations duration=69.47µs
 INFO: [Jul  2 22:42:32.000] creating missing pterodactyl0 interface, this could take a few seconds...
FATAL: [Jul  2 22:42:32.000] failed to configure docker environment error=Error response from daemon: Pool overlaps with other one on this address space

Stacktrace:
Error response from daemon: Pool overlaps with other one on this address space`


Network i only have are:
Local
host
bridge
the panel docker-compose files created a network named `panel_default`
and the wing docker-compose created a new one named `wings0`

i have 1 dedicated server
128GB RAM
550SSD
32cores

im using cloudflared tunnels (im using CLI mode because docker container cloudflared doesnt really work or is not really compatible with all application/containers.)
im hosting everything in the same dedicated server.

i only have a nginx, xbackbones and portainer

what i just noticed if i run the `wings` container  with the `panel` container shutdown the `wings` start working fine they even have their own ports showing and the container seems running without the error but once i turn on the `panel` container the `wing` container gives that error

https://cdn.junny2233.xyz/mafE3/riZuteLU44.png/raw

https://junny2233.link/ZkTGnsUP/panel-docker-compose.yml

https://junny2233.link/jbWAtTKg/wings-docker-compose.yml


thank you.

**Reply by treverehrfurth** [reply]
I have this same problem.... was hoping for some answers by now lol. Did you figure it out?

**Reply by treverehrfurth** [reply]
I got this error previously with the panel, but fixed it by changing the subnet in the panel docker-compose with one that wasn't being used in docker. 
Thought the same would work with the wings agent but i'm getting nothing. I have changed it 4 times and nothing else is using the subnet but it still shows that error

**Comment by TheGinGear** [comment]
So, I, probably like a few of you, was having some issues. I wanted to put this on the same server I was running the reverse proxy from, in the one I created from the guide linked [above.](https://technotim.live/posts/traefik-portainer-ssl/)

Many issues arrived from this due to it needing the same ports. I couldn't open them for it to communicate to the proxy network, and if I manually added the panel, it would need to be readded every update/reboot. You can't even change the network at the bottom of the docker compose to use proxy, as it was ignoring that and creating it's own. What I ended up finding was this: you can add BOTH networks to the docker compose, and specify the panel to connect to the proxy network as well. Here: 
```yaml
...

  panel:
    image: ghcr.io/pterodactyl/panel:latest
    restart: always
    # Commenting this out, as is on the same device (and now network) as the reverse proxy -Gin
    # ports:
    #   - "80:80"
    #   - "443:443"
    links:
      - database
      - cache
    volumes:
      - "/srv/pterodactyl/var/:/app/var/"
      - "/srv/pterodactyl/nginx/:/etc/nginx/http.d/"
      - "/srv/pterodactyl/certs/:/etc/letsencrypt/"
      - "/srv/pterodactyl/logs/:/app/storage/logs"
    environment:
      <<: [*panel-environment, *mail-environment]
      DB_PASSWORD: *db-password
      APP_ENV: "production"
      APP_ENVIRONMENT_ONLY: "false"
      CACHE_DRIVER: "redis"
      SESSION_DRIVER: "redis"
      QUEUE_DRIVER: "redis"
      REDIS_HOST: "cache"
      DB_HOST: "database"
      DB_PORT: "3306"
    # This specifies Panel to join both the default and proxy network - Gin
    networks:
      - default
      - proxy
networks:
  # changing to proxy network here doesn't work - Gin
  default:
    ipam:
      config:
        - subnet: 172.20.0.0/16
  proxy: 
    external: true
```

**Reply by Waltbo** [reply]
how did you setup your wing config/compose? Also did you not have to add any labels to get the panel to work?

**Comment by Adrian-Duque** [comment]
hey, after running "sudo docker-compose up -d --force-recreate" i get these mesage 
"Error response from daemon: error while creating mount source path '/srv/pterodactyl/database': mkdir /srv/pterodactyl: read-only file system" i've also tried "sudo docker-compose up -d" but i get same error

**Comment by MickoLjudinicko** [comment]
Hey, awesome post and great video first and foremost!

I'd like to know if you've discovered a way to easily put game server containers (deployed by the wing agent) behind traefik.

Because everytime I deploy a new game server (minecraft, rust, gmod, etc...), it can only be accessed through the ip address and port, but I'd like to remove the port mapping and instead force it to use the traefik reverse proxy.

Thx in advance!

**Comment by jerembdn** [comment]
Hi, i'm using Traefik for load balancing and pterodactyl, wings on the same machine.
I have this traefik labels in my pterodactyl panel compose :
```

    labels:
      - "traefik.enable=true"
      - "traefik.docker.network=proxy"
      - "traefik.http.services.pterodactyl.loadbalancer.server.port=8080"

      # HTTP -> HTTPS Redirect
      - "traefik.http.routers.pterodactyl.entrypoints=http"
      - "traefik.http.routers.pterodactyl.rule=Host(`pterodactyl.example.com`)"
      - "traefik.http.middlewares.pterodactyl-https-redirect.redirectscheme.scheme=https"
      - "traefik.http.routers.pterodactyl.middlewares=pterodactyl-https-redirect"

      # HTTPS
      - "traefik.http.routers.pterodactyl-secure.entrypoints=https"
      - "traefik.http.routers.pterodactyl-secure.rule=Host(`pterodactyl.example.com`)"
      - "traefik.http.routers.pterodactyl-secure.tls=true"
      - "traefik.http.routers.pterodactyl-secure.service=pterodactyl"

networks:
  proxy:
    external: true
```

I have portainer running in another container and mapped by traefik and it works well (with approximately the same labels config). But Ptero won't work and tells me Bad gateway.. Is it a port mapping issue ?
On the traefik dashboard everything is fine, and nothing in the logs.

**Reply by jerembdn** [reply]
The pterodactyl panel is running fine but is just not exposed on the network or something.. I'm a little bit lost with this..

**Comment by hillyarb** [comment]
I am stuck with a red heart. I have plenty of other things running on my server so 443 was not available. I changed it to 444 on the outside of the container, how would that impact everything that follows? Would the Daemon port need to be 444? What else would be good to double check to make it work?

**Reply by gadget78** [reply]
It's gotta use 443, that's the port that ssl uses...
If you already have other things using your host/server, best way is to install something like nginx proxy manager ...

**Reply by hillyarb** [reply]
Maybe I don't understand how to use the proxy manager. Because nginx proxy manager is what is using port 443. I am using portainer to deploy things, and so had to change the external ports for something to deploy. 
I did check the logs though, and I am getting this specific error: Error response from daemon: Pool overlaps with other one on this address space.
I searched the pterodactyl discord and found someone with this same error, there was a posted solution, I tried that and still having the same problem.

**Reply by g1eagle** [reply]
In your pterodactyl config 
change your ports to something like 
      - "30001:80"
      - "30002:443"

Then make sure to point your proxy to 30001. for Wings.. you might be able to get away with 8443 another SSL port.. but I run wings on a seperate server.

**Reply by g1eagle** [reply]
You might be able to change wings config the same way, then uses the damon port in the interface to point at the port you used in the wings docker config. But honestly I'm just guessing on this.

**Comment by TylerJaacks** [comment]
How do I do this without a reverse proxy? When I follow the steps I have no web ui running on my network?

**Comment by Floslis** [comment]
**Great documentation!**
I am currently experiencing an issue where, upon pressing the 'Launch SFTP' button in the server settings, I log in and receive the error message:

*Permission denied.
Error code: 3
Server error message: Permission denied*

How can this be resolved?

**Comment by g1eagle** [comment]
I finally managed to get this to work.. I had to point my local DNS at Traefik for both Pterodactyl and wings.. Then the green heart showed up.

**Comment by alexbakertech** [comment]
Hey everyone. I followed the documentation as described in the guide with the only difference being that I used the HA-Proxy package in pfSense as my reverse proxy. Setting this up initially, I ran into issues getting the panel to connect to the wings agent. I kept getting a CORS errors and warnings about missing packet headers. 

After some troubleshooting, I turned to google, which led me to a reddit thread that suggested going into my HA-Proxy backend (Services/HAProxy/Backend/Edit) that was handling the wings traffic and simply disabling the Encrypt(SSL) setting in the server list. I did that and sure enough it just started working.

Im guessing it has something to do with not needing a certificate for the daemon connection since its not HTTPS traffic. Whatever the reason, it wasnt at all obvious to me at the time, so I wanted to comment here for next time I need to set this up with this guide. (Oh and I guess for anyone else here that this may help...)

**Reply by TheRealKraken** [reply]
Hi! Nooby linux user here, where exactly is this HAProxy located? Is this Traefik related? Or is this related to the wing server itself?

**Comment by PsychicSubset** [comment]
im not sure if youre still responding but i could use some help, after setting up the config and restarting wins i get this error FATAL: [May 10 10:05:08.045] failed to configure docker environment error=Error response from daemon: Pool overlaps with other one on this address space

Stacktrace:
Error response from daemon: Pool overlaps with other one on this address space

**Comment by TheSimplier** [comment]
For anyone here in 2025. Make sure to update the version on the wings installation. ghcr.io/pterodactyl/wings:v1.6.1 is waaaay out of date. Latest as of today is 1.11.13. 
This will also fix issues with SFTP if you're experiencing those.
